From 1bfd0e015be7dd22a44995dd2a7002328aedc0e6 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Sat, 9 Nov 2013 22:24:37 +0800
Subject: [PATCH] e2fsprogs: fix tests/f_extent_oobounds

Use $DEBUGFS and $MKE2FS to get the in-tree executables
for this test.

(Build machines which run make check shouldn't need to have
e2fsprogs installed, and we should be testing just-built versions
of the tools anyway)

This patch is from:
http://www.spinics.net/lists/linux-ext4/msg38880.html

Eric Sandeen had sent it to the upstream, but haven't been merge by now.

Upstream-Status: Backport

Signed-off-by: Eric Sandeen <sandeen@xxxxxxxxxx>
Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 tests/f_extent_oobounds/script | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/f_extent_oobounds/script b/tests/f_extent_oobounds/script
index 31ac6c9..b00b031 100644
--- a/tests/f_extent_oobounds/script
+++ b/tests/f_extent_oobounds/script
@@ -4,8 +4,8 @@ SKIP_GUNZIP="true"
 TEST_DATA="$test_name.tmp"
 
 dd if=/dev/zero of=$TMPFILE bs=1k count=256 > /dev/null 2>&1
-mke2fs -Ft ext4 $TMPFILE > /dev/null 2>&1
-debugfs -w $TMPFILE << EOF  > /dev/null 2>&1
+$MKE2FS -Ft ext4 $TMPFILE > /dev/null 2>&1
+$DEBUGFS -w $TMPFILE << EOF  > /dev/null 2>&1
 write /dev/null testfile
 extent_open testfile
   insert_node 0 15 100
-- 
1.8.3.1

